src/hooks/useQuery/useQuery.ts

```diff
@@ -1,6 +1,5 @@
 import { useMemo } from 'react';
-import { QueryKey, UseQueryResult, useQuery as useRqQuery } from 'react-query';
-import { UseQueryOptions } from 'react-query/types/react/types';
+import { QueryKey, UseQueryResult, UseQueryOptions, useQuery as useRqQuery } from '@tanstack/react-query';
```

Comment:
In all places we use RQ abbrev. could you use it here as well?

Reply:
I think we only use it when we need to name our abstraction the same way that it's named in react-query, for example:

`useQuery as useRqQuery`

I'm not sure if using more of these abbreviations brings any value but maybe I misunderstood 🤔

Reply:
nono, I meant Rq -> RQ, just it :D 
And its nothing relevant

Reply:
done